src/app/View.ts

```diff
@@ -118,7 +129,25 @@ export class View {
     const containerGame = Render.elementFactory('div', {
       className: 'container-game',
     });
-    //render elements
+
+    rollADiceButton.addEventListener('click', () => {
+      const rollResult = gameController.breed();
+      herdView.innerText = JSON.stringify(
+        newPlayer.theHerd.theAnimals.map(([animal, count]) => [
+          animal.theName,
+          count,
+        ]),
+      );
+      rollResultView.innerText = JSON.stringify(rollResult);
+      gameController.nextPlayer();
+      gameController.startTurn();
```

Comment:
add delay between next player and start calls - to ensure leaving interval in timer

Reply:
I think we could use some refactoring in the Timer component. Actual solution happened to not work well after all. I can only see it now, after using it.